app/views/notifications/_milestone.html.erb

```diff
@@ -0,0 +1,16 @@
+<% json_data = notification.json_data %>
+<div class="content notification-content badge-content">
+  <div class="badge-title">
+    Your post <a href="<%= json_data["article"]["path"] %>"><%= json_data["article"]["title"] %></a> just passed <%= notification.milestone_count %> <%= notification.milestone_type.pluralize.downcase %>!
+  </div>
+  <p class="badge-description milestone-emojis">🎉🎉🎉🎉🎉🎉🎉🎉</p>
+  <a href="https://media.giphy.com/media/<%= json_data["gif_id"] %>/giphy-downsized.gif" rel="noopener noreferrer" target="_blank">
```

Comment:
The `href` here should giphy _page_ as opposed to the file itelf. That would be `https://giphy.com/gifs/<%= json_data["gif_id"] %>` as opposed to the `https://media.giphy.com/media...` link for the image.

Reply:
```suggestion
  <a href="https:/giphy.com/gifs/<%= json_data["gif_id"] %>" rel="noopener noreferrer" target="_blank">
```

Example: https://giphy.com/gifs/y8Mz1yj13s3kI

I think it technically redirects to the full link: `https://giphy.com/gifs/excited-oprah-shouting-y8Mz1yj13s3kI`, which should be okay. Would this need a `data-no-instant` attribute?

Reply:
> Would this need a `data-no-instant` attribute?

Nope Instaclick knows how to deal with external links.